core/themes/basis/color/color.inc

```diff
@@ -0,0 +1,72 @@
+<?php
+
+$info = array(
+  // Available colors and color labels used in theme.
+  'fields' => array(
+    'header' => t('Header background'), // And active pager link.
+    'sitename' => t('Site name background'),
+    'slogan' => t('Slogan background'),
+    'titleslogan' => t('Site name and slogan'), // And main menu links.
+    'bg' => t('Main background'), // And submenu text, pager text, and drop down arrows
+    'footer' => t('Footer background'),
+    'text' => t('Text color'),
+    'link' => t('Link color'),
+    'hovermenu' => t('Hover menu link background'),
+    'border_thingy' => t('Border thingies'), //These are all over the place.
```

Comment:
Lol, how about "Section Divider Color" ?

Reply:
Section border color?

Reply:
I guess that would work. the thing that threw me off was that it's also used in the vertical tabs on edit forms.
![borders](https://user-images.githubusercontent.com/1253691/39479448-1b9e767e-4d1a-11e8-800d-39a7ad570800.png)